cmd/init.go

```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"errors"
 	"fmt"
-
 	"github.com/railwayapp/cli/entity"
+	errors2 "github.com/railwayapp/cli/errors"
```

Comment:
Thoughts on `CliErrors` or `RailwayErrors`?

Reply:
CLIErrors is good. IDE named it `errors2`